internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json

```diff
@@ -19916,12 +19889,6 @@
         "Mean_FamilySize_Household_FamilyHousehold_ForeignBorn_PlaceOfBirthSouthamerica",
         "Mean_FamilySize_Household_FamilyHousehold_ForeignBorn_PlaceOfBirthSouthernEasternEurope",
         "Mean_FamilySize_Household_FamilyHousehold_ForeignBorn_PlaceOfBirthWesternAsia",
-        "Mean_HouseholdSize_Household_ForeignBorn_OccupiedHousingUnit_OwnerOccupied_PlaceOfBirthAfrica",
```

Comment:
@Rohit231998 @manishvats2 Can you pls confirm these deletions from CensusACS5YearSurvey_SubjectTables_S0504 are expected?